### PR TITLE
Accessibility improvements

### PR DESCRIPTION
**Pull request by actlikewill**
https://www.loom.com/share/20931ea7ccd944ff86b850482c20ce50?sid=510213df-d4ce-407d-beca-fc78fb23d294

closes https://github.com/laws-africa/kenyalaw-pj/issues/63

### CODE REVIEW

**Comment by actlikewill** [comment]
@longhotsummer @niiroobiro I've linked a small loom for this pr above

File: peachjam/templates/peachjam/_header.html [hunk 1]
```diff
@@ -1,7 +1,14 @@
 {% load static i18n %}
 <header>
+  {% block skip-links %}
+    <div id="skip-links">
+      <a class="bg-white" href="#top">Go to page content</a>
```
Comment (longhotsummer) [inline peachjam/templates/peachjam/_header.html:5]: Translateable

**Comment by longhotsummer** [comment]
This looks great. The updated colour choices make sense.

The visited colour for lawlibrary isn't the right shade now that the primary colour has changed.

The visited link colour would be better to be declared as a SCSS variable in `variables.scss` near where the primary colour is declared, so that they're nearby to each other.

**Comment by niiroobiro** [comment]
@actlikewill I love the colour changes. Consider adding IDs to "Recent Judgments","Recent legislation" and "Gazettes" This will give users the flexibility to navigate directly to sections that are most relevant to them. Especially when you are using a screen reader.

File: africanlii/static/stylesheets/africanlii.scss [hunk 2]
```diff
@@ -31,3 +31,7 @@ iframe.video {
     display: block;
   }
 }
+
+a:visited {
```
Comment (longhotsummer) [inline africanlii/static/stylesheets/africanlii.scss:35]: is this necessary if peachjam already has it?

File: peachjam/templates/peachjam/_header.html (hunk 1 above)
Comment (actlikewill) [reply peachjam/templates/peachjam/_header.html]: done

File: africanlii/static/stylesheets/africanlii.scss (hunk 2 above)
Comment (actlikewill) [reply africanlii/static/stylesheets/africanlii.scss]: fixed, I also had to set the button color for africanlii to black like we do in other liis

**Comment by actlikewill** [comment]
> @actlikewill I love the colour changes. Consider adding IDs to "Recent Judgments","Recent legislation" and "Gazettes" This will give users the flexibility to navigate directly to sections that are most relevant to them. Especially when you are using a screen reader.

@niiroobiro I was strongly considering this, but at the same time, I'm wary of adding too many links at in the skip links section. They should be few brief in order to be useful, to allow the user to quickly jump to the part of the page they are interested in. Lets wait a bit and we can come back to this when, we do some more focused accessibility stuff.

**Review by longhotsummer — APPROVED** [review]

**Pull request merged**